tests/test_main.py

```diff
@@ -6,18 +6,18 @@
 from app.point import Point
 
 
-# @pytest.mark.timeout(5)
-# def test_deletion():
-#     items = [(f"Element {i}", i) for i in range(1000)]
-#     dictionary = Dictionary()
-#     for key, value in items:
-#         dictionary[key] = value
-#     for key, value in items:
-#         assert dictionary[key] == value
-#     assert len(dictionary) == len(items)
-#     for key, value in items:
-#         del dictionary[key]
-#     print(len(dictionary))
+@pytest.mark.timeout(5)
```

Comment:
where you supposed to make changes to this file?

Reply:
this is a block of code from the tests, I don't have to change it. I already had a conversation with Lyudmila in the qna chat

Reply:
in the condition for the task, it says that if I want to test the delete method, I have to unlock a certain code, so I just unlocked it

Reply:
I don't think you have to push your local changes for testing to remote